config/sync/webform.webform.debt_management_enquiry.yml

```diff
@@ -196,10 +196,10 @@ settings:
   wizard_confirmation: true
   wizard_confirmation_label: ''
   wizard_track: ''
-  preview: 0
+  preview: 2
   preview_label: ''
   preview_title: ''
-  preview_message: ''
+  preview_message: '<span style="-webkit-text-stroke-width:0px; color:#000000; display:inline !important; float:none; font-family:Lato,&quot;Open Sans&quot;,sans-serif; font-size:21px; font-style:normal; font-variant-caps:normal; font-variant-ligatures:normal; font-weight:400; letter-spacing:normal; orphans:2; text-align:start; text-decoration-color:initial; text-decoration-style:initial; text-decoration-thickness:initial; text-indent:0px; text-transform:none; white-space:normal; widows:2; word-spacing:0px">Before sending information, check the information is accurate.&nbsp;</span>'
```

Comment:
Might it be easier/more manageable to use a CSS class set in the wider nidirect theme over inline styles in the webform config?

Reply:
Dodgy cut and paste into the editor field ... will fix